spec/support/shared_examples_for_schedulable.rb

```diff
@@ -3,7 +3,7 @@
 
   it 'gets queued on enqueued_times' do
     enqueued_times.each do |enqueued_time|
-        # update to formatted_enqueue_time when updating sidekiq-cron to 1.9+ (sidekiq 7 support)
+        # TODO: Once on sidekiq-cron v1.9 can updated method to "formatted_enqueue_time"
```

Comment:
Is this just a spelling issue? Unclear what the difference here would be

Reply:
LOLZ! Its def a spelling issue but on the sidekiq-cron side. 

Long story short: 
- sidekiq-cron v1.9 introduces `formatted_enqueue_time` to replace mispelled `formated_enqueue_time` and is not backwards compatible.
- We cannot use sidekiq-cron v1.9  on Rails 4.2 but can on Rails 5 (sidekiq-cron v1.9 is dependent on a version of rack that is not compatible with Rails 4.2)  which is why when I had next version of app, I had to pin the version to anything before v1.9
- After Rails 5 is on prod, we can remove pinning the version for sidekiq-cron so we can get more recent updates as we go,

Reply:
Some decision making I made here, 
- I chose to pin the version of sidekiq-cron as opposed to having yet another gem that uses a different gem on the `next?` logic
       - This is so that we would not have to have 2 versions of tests that use this method (one with the correctly spelled method and one with the typo method on older versions)